packages/scm/src/browser/style/index.css

```diff
@@ -90,9 +90,6 @@
 
 .theia-scm-input-message-container textarea {
     line-height: var(--theia-content-line-height);
-    font-size: var(--theia-ui-font-size1);
```

Comment:
What about classes like `theia-scm-input-message-idle`, `theia-scm-input-message-info` and so on?